mapper/transaction.go

```diff
@@ -172,9 +171,50 @@ func CrossChainTransactions(
 			idx++
 		}
 	default:
-		panic("Unsupported transaction:" + reflect.TypeOf(t).String())
+		return nil, fmt.Errorf("Unsupported transaction: %s", reflect.TypeOf(t).String())
```

Comment:
```suggestion
		return nil, fmt.Errorf("Unsupported transaction: %T", t)
```

Reply:
Also need to remove the `reflect` import since it'll be no longer used after this change